src/customize-pop/dock-settings.md

```diff
@@ -0,0 +1,103 @@
+# Dock Settings
+
+This section describes settings located in the `Dock` tab for modifying Dock visibility behavior and Dock icons.
+
+![Customize Dock](/images/customize-desktop/customize-dock.png)
+
+## Enable Dock
+
+Disable or enable the Dock's visibility.
+
+**Dock Enabled**
+
+![Dock Enabled](/images/customize-desktop/dock-enabled.png)
+
+**Dock Disabled**
+
+![Dock Disabled](/images/customize-desktop/dock-disabled.png)
+
+## Dock Options
+
+Dock options include settings to enable icons for utilities (such as the Launcher, Applications menu, and viewing workspaces), configure focusing behavior when an application's Dock icon is clicked, and enabling visual characteristics of the Dock.
```

Comment:
> such as the Launcher, Applications menu, and viewing workspaces

Not sure why you didn't say `Workspaces menu` for the last item to match the format of the other two.

> Dock options include settings to... enabling visual characteristics of the Dock.

The `enabling` in the last list item should be `enable` just like the first one.